final_prep/README.md

```diff
@@ -44,31 +44,34 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1440" alt="Screen Shot 2022-04-10 at 8 40 04 PM" src="https://user-images.githubusercontent.com/102133027/162656941-bd565e60-0053-4c64-9274-c4bfcb17f78e.png">
+
+- <img width="1440" alt="Screen Shot 2022-04-10 at 8 44 22 PM" src="https://user-images.githubusercontent.com/102133027/162656959-a9701196-420d-4046-96da-85ed462a6fd3.png">
```

Comment:
Looks like a good amount of outside-of-class Turing time is there, as well as those workouts - if that's why you need, GREAT!